### PR TITLE
DDF-2824 update catalog rest endpoint docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates documentation relating to catalog REST endpoint
#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth @clockard 
#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@jlcsmith
@stustison

#### How should this be tested? (List steps with links to updated documentation)

verify content updates

#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/ddf/files/894605/documentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc [hunk 1]
```diff
@@ -27,31 +28,45 @@ The Catalog REST Endpoint provides the capability to query, create, update, and
...
+By default, the response body will include the XML representation of the Metacard.
+
+`<metacardId>` is the `Metacard.ID` of the metacard to be retrieved.
 
 |\${public_url}/services/catalog/<metacardId>
 
 |federated read
 |HTTP GET
-|The SOURCE ID of a federated source is appended in the URL before the ID of the Metacard to be retrieved is appended to the end. (`<sourceId>` is the `FEDERATED SOURCE ID` and `<metacardId>` is the `Metacard.ID` of the Metacard to be retrieved.)
+|The SOURCE ID of a federated source is appended in the URL before the ID of the Metacard to be retrieved is appended to the end.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc]: ... is appended *to* the URL before ... ?

File: distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc [hunk 2]
```diff
@@ -62,3 +77,86 @@ By default, the response body will include the XML representation of the Metacar
...
+      "version" : "2.5.0-SNAPSHOT"
+   }
+]
+----
+
+Note that for all RESTful CRUD commands only one metacard ID is supported in the URL, i.e., bulk operations are not supported.
+
+===== Interacting with the REST CRUD Endpoint
+
+Any web browser can be used to perform a REST read.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc]: not sure it matters but wondering but thought the `w` was capitalized for `Web browser`

File: distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc]: I'm finding conflicting sources for this one. Most of the advice to capitalize comes from older sources, though. It looks a little _self important_ to me, but I'm open to counter-arguments.

**Comment by ricklarsen** [comment]
<kbd>merge?</kbd>

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**